scripts/jira/cloner.py

```diff
@@ -203,30 +220,43 @@ def scan_issue(issue, connection):
         if has_fix_versions_label(issue):
             actions.append(Action(issue.key, "Remove Fix versions label", remove_needs_fix_version_label, issue=issue))
 
+    if get_sprint(issue) is None:
+        actions.append(Action(issue.key, "Issue needs to be included in a sprint", None))
```

Comment:
What if we don't intend to backport a change in the current sprint?

Reply:
If the original issue is in the sprint, the new one better be put in the same sprint. Otherwise, it is forgotten.
We can always manually reshuffle sprints when the issue is on the Agile Board.

Reply:
+1 on what Giga said. Plus, you get the alert from the script about an issue without a sprint, and it will clone bugs using the same sprint than the parent. If needed, you can change it manually afterwards.